### ISSUE TITLE
Combined @Parameter @PathParam 

### ISSUE THREAD

**Issue by rienafairefr** [swagger-api/swagger-core#2818]
This concerns version 2.0.X of swagger-core, when adding a  `@Parameter` annotation to a JAX-RS `@PathParam` method parameter, is it an override, an addition ?

Because this minimal example is failing for me :

```
public class PathParamParameterTest {
    class AnnotatedPathParamResource{
        @DELETE
        @Operation(summary = "Delete user")
        @Path("/{id}")
        public void deleteUser(@Parameter(schema = @Schema(type = "integer", format = "int32"))
                               @PathParam("id") int id) {

        }
    }

    @Test
    public void testAnnotatedPathParam(){
        Reader reader = new Reader(new OpenAPI());
        OpenAPI openAPI = reader.read(AnnotatedPathParamResource.class);
        Assert.assertEquals(openAPI.getPaths().get("/{id}").getDelete().getParameters().get(0).getSchema().getFormat(), "int32");
    }
}
```
The `int` PathParam with an `int` type, without the `@Parameter` generates and int32 format, but when adding a `@Parameter` with an int32 schema it generates an int64 format, this seems really weird.

I feel like there a bug in DefaultParameterExtension in swagger-jaxrs2

**Comment by frantuma** [comment]
fixed in #2819

**Comment by rienafairefr** [comment]
Thanks. From git tags I'm guessing this fix is included in 2.0.2 ?
